app/layout.tsx

```diff
@@ -1,15 +1,18 @@
+"use client";
+
 import React from "react";
-import "./globals.css";
+import { ToastContainer } from "react-toastify";
 import { Inter } from "next/font/google";
 import DefaultContainer from "components/defaultContainer";
 import ReactQueryProvider from "../queries/ReactQueryProvider";
 
-const inter = Inter({ subsets: ["latin"] });
+import "react-toastify/dist/ReactToastify.css";
+import "./globals.css";
+import "assets/styles/font.css";
+import "assets/styles/checkbox.css";
+import "assets/styles/toast.css";
```

Comment:
```suggestion
import "react-toastify/dist/ReactToastify.css";
import "./globals.css";
import "assets/styles/font.css";
import "assets/styles/checkbox.css";
import "assets/styles/toast.css";
```

이부분 `global.css`나 하나로 common.css등으로 모는 건 어떨까요?🥰 혹은 global css를 전역으로 받는 styled component를 잡는것도 좋아보입니다!

Reply:
간단하게 `global.css` 나 `common.css` 로 모아버리겠습니다! 👍